src/sql/workbench/contrib/query/browser/queryEditor.ts

```diff
@@ -267,7 +270,6 @@ export class QueryEditor extends EditorPane {
 
 	private setTaskbarContent(): void {
 		// Create HTML Elements for the taskbar
-		const separator = Taskbar.createTaskbarSeparator();
```

Comment:
this is a HtmlElement object, when we need multiple separators we have to create multiple objects. please compare the differences between the SQL Server toolbars in the PR description